R/epidist.R

```diff
@@ -429,7 +429,7 @@ format.epidist <- function(x, header = TRUE, vb = NULL, ...) {
   } else {
 
     # decide on parameter format from magnitude of number
-    params <- unlist(distributional::parameters(x$prob_dist))
+    params <- parameters(x)
     format_params <- ifelse(
       test = any(params > 9.999e-3 & params < 1e4),
```

Comment:
Presumably this won't break anywhere (e.g. if superspreading has very large MLE for k)?

Reply:
It should work fine. Here's an example of an `<epidist>` object for superspreading (negative binomial) with large and small values for k.

``` r
library(epiparameter)
transmission <- epidist(
  disease = "SARS", 
  epi_dist = "offspring_distribution", 
  prob_distribution = "nbinom", 
  prob_distribution_params = c(mean = 1, dispersion = 1e10)
)
#> Citation cannot be created as either author, year or DOI is missing
transmission
#> Disease: SARS
#> Pathogen: NA
#> Epi Distribution: offspring distribution
#> Study: No citation available
#> Distribution: negbin
#> Parameters:
#>   n: 10000000000.000
#>   p: 1.000

transmission <- epidist(
  disease = "SARS", 
  epi_dist = "offspring_distribution", 
  prob_distribution = "nbinom", 
  prob_distribution_params = c(mean = 1, dispersion = 1e-10)
)
#> Citation cannot be created as either author, year or DOI is missing
transmission
#> Disease: SARS
#> Pathogen: NA
#> Epi Distribution: offspring distribution
#> Study: No citation available
#> Distribution: negbin
#> Parameters:
#>   n: 1e-10
#>   p: 1e-10
```

<sup>Created on 2023-04-05 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
Thanks. Although I notice it seems to be converting (mean,dispersion) to (n,p) in epidist object here?

Reply:
Yes, that's due to the use of the {distributional} package underlying the {epiparameter} print method. We could write a conversion into the print method to make sure the parameters printed are the same as those input into the `<epidist>`, in this case mean and dispersion. 

What do you think?